crates/meilisearch/tests/search/formatted.rs

```diff
@@ -6,14 +6,14 @@ use crate::json;
 
 #[actix_rt::test]
 async fn formatted_contain_wildcard() {
-    let server = Server::new().await;
-    let index = server.index("test");
+    let server = Server::new_shared();
+    let index = server.unique_index();
```

Comment:
We were really that close to be able to use a [shared nested index](https://github.com/meilisearch/meilisearch/blob/6b67f9fc4c04bc07cd38e0ff67faff96b0078299/crates/meilisearch/tests/common/mod.rs#L302), but we can’t because we’re playing with the displayed attributes here.
So sad